### DISCUSSION TITLE
Best way to ignore a constant deployment key?

### DISCUSSION THREAD

**Discussion by robertson-bauercontrols** [EWSoftware/VSSpellChecker#315] [Q&A]
I have an in-code key, which is thousands of characters long.  I'm hoping i can find a way to ignroe the spellchecker for this line of code, without disabling the entire file.

What's the correct pattern to accomplish this?

Thanks so much!

![image](https://github.com/user-attachments/assets/79c5ee8a-bb1e-458a-ac1f-1df3c077bd3a)

**Comment by EWSoftware** [comment]
It looks like you've probably got the options to ignore words in mixed/camel case and words with digits disabled.  You can turn them on unless you really need it to spell check those types of words.  They're on by default to ignore them as such words do tend to introduce more false reports.  Otherwise, just add an Ignore Spelling directive comment to the file to ignore the unwanted misspellings:

```
// Ignore Spelling: Lsb9y Bh
```

**Reply by robertson-bauercontrols** [reply]
Thanks for your response. 

Yeah, the settings I have for ignore fill my needs nicely as of now (aside from this one thing).  The problem with the explicit ignore suggestion is that the spellchecker is interpreting hundreds of mispelled words because the length of the key.

I'm thinking of moving the key into a deployment_keys subdirectory with a spellcheck ignore.  Maybe I can even just add a global setting, and ignore a specific filename pattern, so i don't need to add extra files.
